README.md

```diff
@@ -212,6 +212,16 @@ resources:
         configuration:
           paths:
             "src": "source"
+builder:
+  customTasks:
+    - name: custom-task-name-1
+      beforeTask: standard-task-name
+      configuration:
+        configuration-key: value
+    - name: custom-task-name-2
+      afterTask: custom-task-name-1
+      configuration:
+        configuration-key: value
```

Comment:
Maybe have a different configuration example for the second custom task?

Reply:
Done